servicetalk-transport-netty-internal/src/main/java/io/servicetalk/transport/netty/internal/SslContextFactory.java

```diff
@@ -180,8 +194,7 @@ private static void configureCertificateCompression(SslConfig config, SslContext
             nettySslProvider = forServer ? SslContext.defaultServerProvider() : SslContext.defaultClientProvider();
         }
 
-        // TODO: change once https://github.com/netty/netty/pull/13145 is merged and released
-        if (nettySslProvider != SslProvider.OPENSSL) {
+        if (!SslProvider.isOptionSupported(nettySslProvider, OpenSslContextOption.CERTIFICATE_COMPRESSION_ALGORITHMS)) {
```

Comment:
It's a frequent case that users have an older version of netty in the classpath. Sometimes due to their build issues, sometimes intentionally. 
When netty releases a new feature, we try to use method handle for that and use it only when a correct netty version is in the classpath. 
Example: https://github.pie.apple.com/servicetalk/servicetalk/blob/40d2f02d6a0723f2d3198bdd59862a4f02f044b8/servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/OptimizedHttp2FrameCodecBuilder.java#L43

Reply:
Otherwise, users may be forced to downgrade ST version too.

Reply:
Wouldn't this code only be triggered once someone opts-in to the cert compression feature though?

Reply:
It will at ST level, which doesn't mean users can not downgrade netty version or have a mess in the classpath.
Another vector is that the feature can be enabled by a library behind a scene, not by an app owner.